composer.json

```
@@ -14,12 +14,12 @@
     },
     "require": {
         "php": ">=7.4",
-        "friendsofphp/php-cs-fixer": "v3.54.0",
+        "friendsofphp/php-cs-fixer": "v3.75.0",
         "adamwojs/php-cs-fixer-phpdoc-force-fqcn": "^2.0"
     },
     "require-dev": {
         "roave/security-advisories": "dev-master",
-        "phpstan/phpstan": "^1.10",
+        "phpstan/phpstan": "^2.1",
```

Let's keep it consistent with other packages:

```suggestion
        "phpstan/phpstan": "^2.0",
```